sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/Scan.java

```diff
@@ -65,6 +65,10 @@ default String description() {
    * exception, data sources must overwrite this method to provide an implementation, if the
    * {@link Table} that creates this scan returns {@link TableCapability#BATCH_READ} support in its
    * {@link Table#capabilities()}.
+   * <p>
+   * If the scan supports runtime filtering and implements {@link SupportsRuntimeFiltering},
```

Comment:
@holdenk @sunchao @viirya, added this comment to clarify the behavior change as discussed.